services/github/github-checks-status.tester.js

```diff
@@ -0,0 +1,49 @@
+'use strict'
+
+const t = (module.exports = require('../tester').createServiceTester())
```

Comment:
similarly we can use the same helpers in the tests
```suggestion
const t = (module.exports = require('../tester').createServiceTester())
const { isBuildStatus } = require('../build-status')
```